### PR TITLE
355 empty meal is being created in admin UI

### LINKED ISSUE #355: Empty meal is being created in admin-ui
In admin-ui, while creating meal I have noticed that,  even when all the fields are empty and I click on 'Save' button. It shows 'the element is created'.

Steps to Reproduce:

1. Login to "http://localhost:3000/#/login"
2. Clicked on 'Create' button, to create new meal.
3. Left all the fields empty and clicked 'Save' button

Expected: Error should be thrown to pass mandatory values
Actual :  Popup with text Meal plan created is displayed

### PR DESCRIPTION
**Pull request by RLCASTRO**
**Describe the technical changes contained in this PR**
Added a simple form validation for creating new meals in the admin-ui application

**Previous behaviour**
Admin was able to create new meals without providing any inputs for any fields in the form to create a new meal.

**New behaviour**
Admin must provide minimum input to be able to create a new meal. The fields with validation are:

* nameEn

**Related issues addressed by this PR**
Fixes #355

### CODE REVIEW

File: admin-ui/src/Meals/MealForm.tsx [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
 
 export const MealForm = () => {
   return (
-    <SimpleForm>
-      <TextInput source="code" fullWidth />
-      <TextInput source="nameEn" fullWidth />
-      <TextInput source="nameFr" fullWidth />
+    <SimpleForm >
+      <TextInput source="code" fullWidth validate={required()} />
+      <TextInput source="nameEn" fullWidth validate={required()} />
```
Comment (shanthisa) [inline admin-ui/src/Meals/MealForm.tsx:7]: This is correct. We need to make sure the meal plan name in English is a required field. Please check the database and mark only the ones marked as 'not null' to be required. For example: name_Fr is an optional field.

**Review by shanthisa — CHANGES_REQUESTED** [review]
You have taken the right approach. Just need to mark only the required fields as required based on the database

File: admin-ui/src/Meals/MealForm.tsx (hunk 1 above)
Comment (RLCASTRO) [reply admin-ui/src/Meals/MealForm.tsx:7]: Done, left validation for nameEn only, according to the database

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@
 .vscode
 node_modules
 .envrc
+admin-ui/package-lock.json
+mealplanner-ui/src/pages/MealPlans/__generated__/MealPlansQuery.graphql.ts
```
Comment (shanthisa) [inline .gitignore]: Should not git ignore generated file. Please commit this file.

**Review by shanthisa — CHANGES_REQUESTED** [review]
Please check in generated types. Otherwise any developer has to run 'npm run relay' everytime.

File: .gitignore (hunk 2 above)
Comment (RLCASTRO) [reply .gitignore]: Removed files from the .gitignore. Thank you

**Comment by shanthisa** [comment]
There seems to be conflict in the graphql.ts generated. Can you please rebase v2 and resolve the conflict? Then it will be good to merge.

**Review by shanthisa — DISMISSED** [review]
LGTM 👍

**Comment by shanthisa** [comment]
This has conflicts. Please resolve

File: .gitignore (hunk 2 above)
Comment (shanthisa) [reply .gitignore]: I am still seeing that the generated graphql and package-lock.json as part of gitignore

**Review by shanthisa — CHANGES_REQUESTED** [review]
There are 37 files which are all the migrations of the past and favorite meal feature changes which are not part of this ticket. Looks like you have many more files merged into your branch and hence it is wrongly reflecting as your changes. Please take a clean clone of v2 branch and make your changes over it and commit and create a pull request.

File: .gitignore (hunk 2 above)
Comment (RLCASTRO) [reply .gitignore]: sorry, I believe is fixed now. Thank you.

File: admin-ui/src/Meals/MealList.tsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import {
   ShowButton,
   TextField,
 } from "react-admin";
+import { Link } from "react-router-dom";
```
Comment (shanthisa) [inline admin-ui/src/Meals/MealList.tsx]: Why is this required in this PR?

**Review by shanthisa — CHANGES_REQUESTED** [review]

File: admin-ui/src/Meals/MealList.tsx (hunk 3 above)
Comment (RLCASTRO) [reply admin-ui/src/Meals/MealList.tsx]: solved !

File: admin-ui/package.json [hunk 4]
```diff
@@ -26,6 +26,6 @@
     "@types/react-dom": "^18.0.0",
     "@vitejs/plugin-react": "^1.3.0",
     "typescript": "^4.6.3",
-    "vite": "^2.9.17"
+    "vite": "^2.9.16"
```
Comment (shanthisa) [inline admin-ui/package.json]: Should not be downgraded

**Review by shanthisa — CHANGES_REQUESTED** [review]

File: admin-ui/package.json (hunk 4 above)
Comment (RLCASTRO) [reply admin-ui/package.json]: solved

**Review by shanthisa — APPROVED** [review]
LGTM 👍

**Pull request merged**